hydra-node/golden/Message SimpleTx.json

```diff
@@ -1,31 +1,28 @@
 {
     "samples": [
         {
-            "peer": {
-                "hostname": "0.0.4.59",
-                "port": 2142
-            },
+            "nodeId": "\u0012Y~𨄈c愱(𤪼\r\u0008l𩦊t`\"􂙨o\u0019nᓚ\u0003ez\u0005㗵ᖐ\n",
```

Comment:
Our arbitrary instance here is maybe a bit annoying. Maybe something like `Text.pack <$> listOf (chooseEnum ('a', 'z'))` yields better readable random values?